src/vimageHosting.cpp

```diff
@@ -0,0 +1,680 @@
+#include "vimageHosting.h"
+#include "utils/vutils.h"
+#include "vedittab.h"
+
+extern VConfigManager *g_config;
+
+VGithubImageHosting::VGithubImageHosting(VFile *p_file, QWidget *p_parent)
+    : m_file(p_file),
+      parent(p_parent)
+{
+    reply = Q_NULLPTR;
+    imageUploaded = false;
+}
+
+VWechatImageHosting::VWechatImageHosting(VFile *p_file, QWidget *p_parent)
```

Comment:
Please group functions of `VGithubImageHosting` together and functions of `VWechatImageHosting` together.